ch_pipeline/analysis/flagging.py

```diff
@@ -68,7 +68,9 @@ class RFIFilter(task.SingleTask):
     threshold_mad = config.Property(proptype=float, default=6.0)
     use_draco_container = config.Property(proptype=bool, default=False)
 
-    def process(self, data) -> Union[containers.RFIMask, dcontainers.RFIMask]:
+    def process(
+        self, data
```

Comment:
Should we include a type hint here as well?
```suggestion
        self, data: Union[ch_util.andata.CorrData, dcontainers.TODContainer, dcontainers.SiderealContainer]
```
Even if not, we should update the docstring for the `data` parameter, below

Reply:
Good call, I'll make that change